setup.py

```diff
@@ -9,6 +9,8 @@
 
 from setuptools import setup, find_packages
 
+# from torch.utils.cpp_extension import BuildExtension, CUDAExtension
```

Comment:
nit: can you please delete these commented lines

Reply:
deleted